SnapshotTests/VisitorCodeViewControllerDynamicTypeFontTests.swift

```diff
@@ -0,0 +1,114 @@
+@testable import GliaWidgets
+import SnapshotTesting
+import XCTest
+
+// swiftlint:disable type_name
+class VisitorCodeViewControllerDynamicTypeFontTests: SnapshotTestCase {
+    static let extra3LargeFontStrategy = Snapshotting<UIViewController, UIImage>.image(
```

Comment:
Could it be moved to `SnapshotTestCase`, because it's duplicated in each new test class?

Reply:
Good idea. Introduced dedicated strategies 40a9070ea6ec0454458589febfe02a2e1697477c.